C++/include/algorithm/sorting/README.md

```diff
@@ -0,0 +1,129 @@
+# Sorting
+Sorting algorithms are designed to arrange data in a particular order. Sorting algoritms have a few properties:
+- #### 1. Stable and Unstable Sorting:
+      Stable: identical elements remain in the same order as before sorting.
+      Unstable: identical elements may be sorted in any order.
+- #### 2. Internal and External Sorting:
```

Comment:
I don't think it's worth introducing the concept of internal versus external sorting algorithms if you don't reference the concept again with regards to specific algorithms.